tools/cxbe/Xbe.cpp

```diff
@@ -411,6 +405,17 @@ Xbe::Xbe(class Exe *x_Exe, const char *x_szTitle, bool x_bRetail, const std::vec
                     m_SectionHeader[v].dwSizeofRaw = RoundUp(r + 2, 4);
                 }
 
+                // calculate virtual size
+                if(v < m_Header.dwSections - 1)
+                    m_SectionHeader[v].dwVirtualSize =
+                        x_Exe->m_SectionHeader[v + 1].m_virtual_addr -
+                        x_Exe->m_SectionHeader[v].m_virtual_addr;
```

Comment:
I've never really noticed how weird and seemingly unnecessary this code is - unless it causes some breakage I'm not seeing rn we should imho just remove this part entirely.

Reply:
I wouldn't be opposed to making this a separate issue/pr.

In my opinion, there's a lot more of cxbe that should be simplified to trust the generated PE from the linker.